poetry/console/commands/export.py

```diff
@@ -11,7 +11,13 @@ class ExportCommand(Command):
     description = "Exports the lock file to alternative formats."
 
     options = [
-        option("format", "f", "Format to export to.", flag=False),
+        option(
+            "format",
+            "f",
+            "Format to export to.",
+            flag=False,
+            default=Exporter.ACCEPTED_FORMATS[0],
```

Comment:
It would be good if we are less dynamic about this since we are documenting `requirements.txt` explicitly as the default. Might be worthwhile to split the format into constants for now.

Reply:
I was actually torn whether this was worth adding a constant or not.  Thanks for the second opinion; I just updated the MR accordingly.